lib/torrent.js

```diff
@@ -323,6 +323,13 @@ class Torrent extends EventEmitter {
       })
     }
 
+    // add BEP09 peer-address
+    // TODO: It should only be included if the client can discover its public IP address and determine its reachability.
+    if (this.peerAddress) {
+      const peers = Array.isArray(this.peerAddress) ? this.peerAddress : [this.peerAddress]
```

Comment:
Upon seeing this, I think we should change the `magnet-uri` API so that `peerAddresses` is always an array, since that's what the plural name indicates. If the user wants the true raw value, they can access `x.pe`, but the `peerAddresses` API is ours so we can make it friendlier if we want. And it saves everyone from needing to do this `isArray` check.

Reply:
Changed! 👍